crates/astria-sequencer/src/app.rs

```diff
@@ -89,6 +92,16 @@ pub(crate) struct App {
     // `prepare_proposal`, and re-executing them would cause failure.
     is_proposer: bool,
 
+    // This is set to the executed hash of the proposal during `process_proposal`
+    //
+    // If it does not match the hash given during begin_block, then we clear and
+    // reset the execution results cache + state delta. Transactions are reexecuted.
+    // If it does match we utilize cached results to reduce computation.
+    //
+    // Resets to null hash at the beggining of prepare_proposal, and process_proposal if
```

Comment:
```suggestion
    // Resets to the default hash at the beginning of prepare_proposal, and process_proposal if
```

Reply:
probably want to specify what the default hash is

Reply:
I don't think it really matters what the default hash is? It's default, anything more is client library implementation detail?